server/app/controllers/ProductionController.ts

```diff
@@ -0,0 +1,74 @@
+import { inject } from 'inversify';
+import TYPES from '../constants/types';
+import { ProductionService } from '../services/ProductionService';
+import {
+  controller,
+  httpDelete,
+  httpGet,
+  httpPatch,
+  httpPost,
+  results,
+} from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { IProduction } from './../models/ProductionModel';
+import { Request } from 'express';
+
+@controller('/productions')
+export class ProductionController extends BaseController {
+  constructor(@inject(TYPES.ProductionService) private productionService: ProductionService) {
+    super();
+  }
+
+  // @desc        Get all production
+  // @route       GET /productions
+  // @access      Public
+  @httpGet('/')
+  public async get(): Promise<results.JsonResult> {
```

Comment:
Can you add the `@Doc` like the other controllers have?

Reply:
of course, forgot